lib/constants.ts

```diff
@@ -30,4 +30,6 @@ export const REFERRER_PATH_STORAGE_KEY = 'referrerPath';
 // Code challenge: Base64 URL-encoded SHA-256 hash of the code verifier.
 export const MIN_VERIFIER_LENGTH = 43;
 export const MAX_VERIFIER_LENGTH = 128;
-export const DEFAULT_CODE_CHALLENGE_METHOD = 'S256';
\ No newline at end of file
+export const DEFAULT_CODE_CHALLENGE_METHOD = 'S256';
+
+export const IDX_API_VERSION = '1.0.0';
```

Comment:
I was thinking should we move constants to each sub-module? not sure if it's a good pattern ...

Reply:
I'm on the borderline with this as well. They were scattered and then were consolidated. I considered and briefly added a constants file to idx, but decided to continue with consolidation. I can be persuaded by a good argument either way.